jerry-core/ecma/operations/ecma-function-object.c

```diff
@@ -1417,6 +1464,34 @@ ecma_op_function_try_to_lazy_instantiate_property (ecma_object_t *object_p, /**<
 
     return NULL;
   }
+
+  if (ecma_compare_ecma_string_to_magic_id (property_name_p, LIT_MAGIC_STRING_NAME))
+  {
+    ecma_extended_object_t *ext_func_p = (ecma_extended_object_t *) object_p;
+    if (!ECMA_GET_SECOND_BIT_FROM_POINTER_TAG (ext_func_p->u.function.scope_cp))
+    {
+      /* Set tag bit to represent initialized 'name' property */
+      ECMA_SET_SECOND_BIT_TO_POINTER_TAG (ext_func_p->u.function.scope_cp);
+      const ecma_compiled_code_t *bytecode_data_p = ecma_op_function_get_compiled_code (ext_func_p);
+
+      ecma_value_t value = *ecma_compiled_code_resolve_function_name (bytecode_data_p);
+      if (value != ECMA_VALUE_EMPTY)
```

Comment:
Isn't there a special constant for this?

Reply:
I've changed it to ECMA_VALUE_EMPTY, that helped me to reduce the EMPTY/ANONYMOUS checks in snapshot the snapshot tool.